fs.go

```diff
@@ -741,6 +741,12 @@ type DeepFS struct {
 	// The root filepath on disk.
 	Root string
 
+	// Set a custom separator for marking when traversing into an archive.
+	// If empty, the default system separator is used.
+	// If set, use DeepFS.WalkDir instead of fs.WalkDir to handle the custom
+	// separator properly
+	InnerFsysSeparator string
```

Comment:
How about just:

```suggestion
	Separator rune
```

(Any reason a whole string is needed?)

Reply:
It could be a single rune. I had originally done it as a string to be flexible for weird(er) use cases. I'll go ahead and swap it to a rune though.

I can also change it to `Separator`, but I didn't want someone to think that setting it would change every separator in the path. 

Like for `/foo/bar.zip/some/inner/path`, setting it to `:` would turn it to `/foo/bar.zip:some/inner/path`, not `:foo:bar.zip:some:inner:path`.

What is your preference:

1. Change it to `Separator`
2. Keep it `InnerFsysSeparator`
3. Rename it to something like `FsysBoundrySeparator` (or some other name)

^\_^

Reply:
Right, okay, so how about `InnerPathSeparator`?